address/address_test.go

```diff
@@ -56,3 +59,30 @@ func TestBech32(t *testing.T) {
 	}
 
 }
+
+func Test_Blech32EncodeDecode(t *testing.T) {
```

Comment:
Please, maintain the same structure of the base58 and bech32 tests. This should be split into `TestFromBlech32` and `TestToBlech32`.